lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -120,21 +120,29 @@ module.exports = function(self, options) {
             }
 
             if (info.extension === 'jpg') {
-              const data = exif.parseSync(file.path);
-              console.log(data);
-              info.description = data.ImageDescription ? data.ImageDescription.trim() : null;
-
-              if (data.Make && data.Model) {
-                info.camera = data.Make.trim() + ' ' + data.Model.trim();
-              } else if (data.Model) {
-                info.camera = data.Model.trim();
-              }
-
-              info.captureDate = data.DateTime ? data.DateTime : null;
-              info.credit = data.Copyright ? data.Copyright.trim() : null;
+              exif.parse(file.path, (err, exifData) => {
```

Comment:
ABR (Always Be Returning) when you invoke a callback-based function like this, so you don't get confused later and think you can put code to run "after this"... after this.